### ISSUE TITLE
Release download blocked as a virus

### ISSUE THREAD

**Issue by tgraupmann** [tgraupmann/WinForm_Ollama_Copilot#2]
Chrome/Edge and maybe Windows defender blocks the setup download falsely flagging as a virus for being an uncommon install.

**Comment by giandre** [comment]
Yes, sadly, I am encountering this as well. I have tried everything I know to force download the file. Do you know of any workaround?

**Comment by tgraupmann** [comment]
Looks like I just have to sign the installer - https://stackoverflow.com/questions/76248724/windows-saying-my-msi-installer-is-a-virus

**Comment by giandre** [comment]
Great. let us know when you have done that. This tool has a lot of promise!

**Comment by tgraupmann** [comment]
I'm looking for a cheaper option than $798 + Shipping for 3 years for code signing certs for all my open source projects.
https://www.sectigo.com/ssl-certificates-tls/code-signing

Self-signed certs did not work.

**Comment by tgraupmann** [comment]
I'm going to switch back to Innosetup which has a warning but it won't be blocked. CA cert fees are insane!

**Comment by tgraupmann** [comment]
I switched to an InnoSetup installer. There's a warning to accept that I didn't use a code signing certificate, but it's not blocked as a virus.

**Comment by tgraupmann** [comment]
Windows started blocking the new installer as a virus again. Back to code signing.
